lib/cartopy/io/ogc_clients.py

```diff
@@ -122,7 +122,8 @@ def _warped_located_image(image, source_projection, source_extent,
     else:
         # Convert Image to numpy array (flipping so that origin
         # is 'lower').
-        img, extent = warp_array(np.asanyarray(image)[::-1],
+        # convert to RGBA to keep the color palette
+        img, extent = warp_array(np.asanyarray(image.convert('RGBA'))[::-1],
```

Comment:
Does adding this change how RGB (without alpha) is read/used?

Below this code (Line 140) there are tests for the shape of img if there is an alpha layer or not. I assume with this change it would always return an RGBA img and not go through the other branch?

Reply:
Do you mean 'test' as in unit test, or as in the conditional below, i.e., is this PR complete now?

Reply:
I meant the conditional.
The PR is complete.

Reply:
I just added some comments.